src/PushNotifications/Subscriptions/FireBaseSubscriptionAppService.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Elders.Cronus.DomainModeling;
+using PushNotifications.Contracts.FireBaseSubscriptions.Commands;
+
+namespace PushNotifications.Subscriptions
+{
+    public class FireBaseSubscriptionAppService : AggregateRootApplicationService<FireBaseSubscription>,
+        ICommandHandler<SubscribeUserForFireBase>,
+        ICommandHandler<UnSubscribeUserFromFireBase>
+    {
+        public void Handle(SubscribeUserForFireBase command)
+        {
+            FireBaseSubscription sub;
+            if (Repository.TryLoad(command.Id, out sub))
```

Comment:
looks like the subscribe method and the constructor of FireBaseSubscription are doing the same thing

Reply:
why is there a subscribe method?